subprojects/common/graphite/src/main/kotlin/com/avito/android/graphite/Graphite.kt

```diff
@@ -0,0 +1,18 @@
+package com.avito.android.graphite
+
+/**
+ * https://graphite.readthedocs.io/en/latest/terminology.html#term-value
+ */
+fun graphiteSeriesElement(value: String) = value.replace(invalidSymbols, "_")
```

Comment:
internal?

Reply:
nope, it's intended to use in clients

Reply:
Removed these functions. It was a copy from statsd module